### DISCUSSION TITLE
Q: ip归属地不显示 || Q: ip attribution is not displayed

### DISCUSSION THREAD

**Discussion by 2176608** [walinejs/waline#2020] [Q&A] [answered]
### Describe the bug | 问题描述

如图：
![image](https://github.com/walinejs/waline/assets/90504717/547f8494-704f-4986-bef1-bfab36dcc67d)

### Website URL | 问题网站

2176608.github.io

### Where your waline deploy? | 服务部署在哪里？

Deta

### Where your comment data store? | 数据存储在哪里？

GitHub

---
### Describe the bug | Problem description

As shown in the picture:
![image](https://github.com/walinejs/waline/assets/90504717/547f8494-704f-4986-bef1-bfab36dcc67d)

### Website URL | Problem Website

2176608.github.io

### Where your waline deploy? | Where is the service deployed?

Deta

### Where your comment data store? | Where is your comment data store?

GitHub

**Comment by lizheming** [comment, marked as answer]
Deta 拿不到真是的用户 IP，建议关闭该功能

---
Deta can't get the real user IP, it is recommended to turn off this function

**Reply by 2176608** [reply]
感谢回答！


---
Thanks for answering!
